### PR TITLE
Session3

### PR DESCRIPTION
**Pull request by enzomatela**
I still can't add juliuscnicolas, and kevinatgithub as reviewer.

### CODE REVIEW

File: Services/ItemServices.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public class ItemServices
+    {
+        public int GetAll(int Id)
+        {
+            return 0;
+        }
+
+        public void Save(ItemDTO itemDTO)
+        {
+
```
Comment (jegtugado) [inline Services/ItemServices.cs:16]: need to map ItemDTO to an Item

**Review by jegtugado — CHANGES_REQUESTED** [review]
Looks good. You only need to add the mapping left.

File: Services/ItemServices.cs (hunk 1 above)
Comment (enzomatela) [reply Services/ItemServices.cs]: Done. Kailangan pa imap yan? understanding ko sa item6 e kailangan lang imap before ipasa sa itemservices.

Comment (jegtugado) [reply Services/ItemServices.cs]: Hi sir need po sa item 8. 
image

**Review by jegtugado — APPROVED** [review]

**Pull request merged**